friends/.gitignore

```diff
@@ -0,0 +1,21 @@
+# See https://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
@algorithmsrock For as long as you plan on using VIM as your text editor while developing software, it is **_your responsibility_** to add the VIM resource file types to your .gitignore file, e.g.

*.swo
*swp
*.swn
*.swm

Not only does no one else on your team need them, if, in fact, someone else on your team is also using VIM, the failure to ignore them will cause your teammate problems.

Do you remember the difficulty not including these file extensions caused you a coupe of weeks prior? I recall I spent a good solid hour of my life resolving the issue with you and teaching you how to avoid it in the future. Heed my advice in this regard.

Lastly, it is sufficient to include VIM resource file type extension exclusions in the .gitignore file in the parent level directory of your project.